server/server.js

```diff
@@ -15,19 +18,21 @@ mongoose
     useUnifiedTopology: true,
     useCreateIndex: true,
     dbName: "Dashboard",
+    useFindAndModify: false,
   })
   .then(() => console.log("Connected to MongoDB"))
   .catch((err) => console.log(`${err}: Failed to connect to MongoDB`));
 
 // default all api endpoints to the defined routes in `api.js`
 const app = express();
 app.use(express.json());
-app.use("/api", require("./api.js"));
 
 app.use(bodyParser.json());
 app.use(bodyParser.json({ type: "text/*" }));
-app.use(bodyParser.urlencoded({ extended: false }));
-
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use("/api", api);
```

Comment:
I think you had a good point with separating the endpoints rather than shove it all in `api.js`! Feel free to get rid of it completely